scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/TaskFilterCriteria.java

```diff
@@ -102,16 +102,16 @@ public void setUser(String user) {
         this.user = user;
     }
 
-    public List<SortParameter<TaskSortParameter>> getSortParameters() {
+    public List<SortSpecifierContainer> getSortParameters() {
         if (sortParameters != null)
             return Collections.unmodifiableList(sortParameters);
         else
             return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1168)